src/mount/mastercomm.cc

```diff
@@ -205,6 +205,7 @@ void master_stats_add(uint8_t id,uint64_t s) {
 static inline void setDisconnect(bool value) {
 	std::unique_lock<std::mutex> fdLock(fdMutex);
 	disconnect = value;
+	masterServerIsDisconnected = value;
```

Comment:
Please consider renaming this variable to isDisconnectedFromMaster or isConnectedToMaster, which is usually recommended for boolean variables.

Reply:
Done